extension/src/background.js

```diff
@@ -453,41 +453,25 @@ chrome.webNavigation.onDOMContentLoaded.addListener(detail => {
 
 // chrome.tabs.onReplaced.addListener(updateState);
 
-// $FlowFixMe
-chrome.tabs.onUpdated.addListener(defensify(async (tabId: number, info, tab: chrome$Tab) => {
-    // too spammy in logs
-    delete tab.favIconUrl
-    delete info.favIconUrl
-    //
-    console.debug('onUpdated %o %o', tab, info)
+// $FlowExpectedError
```

Comment:
yeah, looks like they are incomplete for these  https://github.com/Shraymonks/flow-interfaces-chrome/issues/1#issue-145335503